.changeset/fuzzy-buttons-beg.md

```diff
@@ -0,0 +1,2 @@
+---
+---
```

Comment:
Just one minor thing to mention... as this becomes public, it would be a good idea to mark it as a `major` version. Probably can be done after you integrate theming into it :)

Reply:
i actually marked it as a major verson in the previous PR so i think that should apply there!